CharacterCreation/01_04_Species_Opaleites.txt

```diff
@@ -14,6 +25,12 @@ underwater sounds. Keeping the second set closed does hurt an Opaleite's speed
 while swimming. They are also used to drain the lungs when making the switch 
 from water to air.
```

Comment:
The sentence fragments in the above par. are jarring, not helpful. We'll want to fix that before ship.